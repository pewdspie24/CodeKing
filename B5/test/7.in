62
10 14
6 13
1 11
8 15
4 13
1 12
9 11
2 13
6 11
10 15
9 12
8 10
10 11
2 12
4 12
6 17
2 14
9 15
10 11
7 17
8 10
10 15
7 12
7 16
1 13
4 18
4 10
1 18
6 10
1 17
3 10
9 17
1 14
5 13
5 17
10 18
10 12
6 12
9 12
8 10
5 13
2 12
4 11
2 18
1 16
3 18
5 10
8 17
1 12
2 17
9 15
5 12
4 11
5 14
2 15
5 14
8 13
10 10
3 14
9 16
8 12
9 12
