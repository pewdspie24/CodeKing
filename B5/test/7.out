100000000000000000
16926659444736
22876792454961
30517578125
78364164096
31381059609
282429536481
2048
43046721
4747561509943
16384
4398046511104
78364164096
34522712143931
2541865828329
59049
1594323
177147
14348907
9765625
1000000000
1000000000000000
505447028499293771
1000000000000000
2251799813685248
4177248169415651
177147
16384
